nixos/modules/services/hardware/xow.nix

```diff
@@ -0,0 +1,31 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.hardware.xow;
+in {
+  options.services.hardware.xow = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        Whether to enable xow or not.
+      '';
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    services.hardware.uinput.enable = true;
+
+    systemd.services.xow = {
+      wantedBy = [ "multi-user.target" ];
```

Comment:
You should use the upstream service definition via `systemd.packages` and only override those values that are wrong/missing.